core/src/main/resources/AppConstants.properties

```diff
@@ -154,6 +154,9 @@ cleanup.database.cron=0 45 1 * * ?
 # maximum number of rows to delete from ibisstore table in one execution of job. Set to 0 for unlimited.
 cleanup.database.maxrows=25000
 
+# cron pattern to be used by cleanup the lock table
+cleanup.lock.cron=0 0/10 * ? * * *
```

Comment:
Is een cleanup van  één keer per uur niet voldoende?

Reply:
Ik heb hier een beetje een voorschot genomen op dat #1502 geïmplementeerd zal worden, maar in dat geval kan een nieuwe default gezet worden of als vaker opschonen echt nodig is de waarde door de ontwikkelaar overschreven worden in de properties van de betreffende Frank.

Enige nadeel dat ik aan 1x per uur zie, ook met de huidige minimumretentie van 1 uur, is dat de effectieve retentie van een job flink oploopt als er iets fout gaat met de betreffende job net nadat de opschoning heeft gelopen. Dan kan dat zomaar op een retentie van 2 uur uitkomen (1 uur ingesteld, 1 uur wachten op de cleanup). Voor jobs die niet zo vaak lopen is dat niet zo erg, maar voor jobs die vaker lopen kan het een ongewenste verdubbeling van retentie zijn.